package.json

```diff
@@ -19,14 +19,27 @@
     "test": "jest --passWithNoTests",
     "tsc": "tsc --project ."
   },
+  "dependencies": {
```

Comment:
I really don't want all of these to become peer dependencies. I'd rather let them end up in prod deps, then let the user install all of these. This is fine because we are focusing on BE services now.